backend/app/src/main/java/com/moh/phlat/backend/controller/ProcessDataController.java

```diff
@@ -392,13 +448,215 @@ public ResponseEntity<ResponseMessage> plrLoad(@PathVariable Long controlTableId
+				break;
+			case "busOwnerTypeOther":
+				_processData = processDataRepository.findAllDistinctBusOwnerTypeOther(controlTableId);
+				break;
+			case "facBuildingName":
+				_processData = processDataRepository.findAllDistinctFacBuildingName(controlTableId);
+				break;
+			case "facilityHdsDetailsAdditionalInfo":
+				_processData = processDataRepository.findAllDistinctFacilityHdsDetailsAdditionalInfo(controlTableId);
+				break;
+			case "physicalAddr1":
+				_processData = processDataRepository.findAllDistinctPhysicalAddr1(controlTableId);
+				break;
+			case "physicalAddr2":
+				_processData = processDataRepository.findAllDistinctPhysicalAddr2(controlTableId);
+				break;
+			case "physicalAddr3":
+				_processData = processDataRepository.findAllDistinctPhysicalAddr3(controlTableId);
+				break;
+			case "physicalAddr":
+				_processData = processDataRepository.findAllDistinctPhysicalAddr4(controlTableId);
+				break;
+			case "physicalCity":
+				_processData = processDataRepository.findAllDistinctPhysicalCity(controlTableId);
+				break;
+			case "physicalProvince":
+				_processData = processDataRepository.findAllDistinctPhysicalProvince(controlTableId);
+				break;
+			case "physicalPcode":
+				_processData = processDataRepository.findAllDistinctPhysicalPcode(controlTableId);
+				break;
+			case "physicalCountry":
+				_processData = processDataRepository.findAllDistinctPhysicalCountry(controlTableId);
+				break;
+			case "physAddrIsPrivate":
+				_processData = processDataRepository.findAllDistinctPhysAddrIsPrivate(controlTableId);
+				break;
+			case "mailAddr1":
+				_processData = processDataRepository.findAllDistinctMailAddr1(controlTableId);
+				break;
+			case "mailAddr2":
+				_processData = processDataRepository.findAllDistinctMailAddr2(controlTableId);
+				break;
+			case "mailAddr3":
+				_processData = processDataRepository.findAllDistinctMailAddr3(controlTableId);
+				break;
+			case "mailAddr4":
+				_processData = processDataRepository.findAllDistinctMailAddr4(controlTableId);
+				break;
+			case "mailCity":
+				_processData = processDataRepository.findAllDistinctMailCity(controlTableId);
+				break;
+			case "mailBc":
+				_processData = processDataRepository.findAllDistinctMailBc(controlTableId);
+				break;
+			case "mailPcode":
+				_processData = processDataRepository.findAllDistinctMailPcode(controlTableId);
+				break;
+			case "mailCountry":
+				_processData = processDataRepository.findAllDistinctMailCountry(controlTableId);
+				break;
+			case "mailAddrIsPrivate":
+				_processData = processDataRepository.findAllDistinctMailAddrIsPrivate(controlTableId);
+				break;
+			default:
+				return ResponseEntity.status(HttpStatus.OK).body(new ResponseMessage("Error", 404, "Column not found.", 
+						_processData));
+		}
+
+		return ResponseEntity.status(HttpStatus.OK).body(new ResponseMessage("success", 200, "Retrieved the items in the column.", 
```

Comment:
Just thinking if we need to send this message - "Retrieved the items in the column". Could you check at other places, in case of success, what are we sending?

Reply:
I think I had checked hence the message.  Will recheck and see what is being done elsewhere.